### PR TITLE
Proper error handling

### PR DESCRIPTION
**Pull request by marekkirejczyk**

### CODE REVIEW

File: src/app/modals/CreateAccount.tsx [hunk 1]
```diff
@@ -19,32 +19,59 @@ enum CreationStatus {
...
-    setMessage(`⏳ Minting ${amount} tokens to ${account.getAddress().toShortString()}`)
-    await faucet(account.getAddress(), 1234n)
-
+    setAccount(account);
+    mintMutation.mutate(account);
+    return account;
+  };
 
+  const mintTokens = async (account: AccountWalletWithPrivateKey) => {
+    await faucet(account.getAddress(), 1234n);
```
Comment (akonior) [inline src/app/modals/CreateAccount.tsx:41]: Extract `1234n` to a contant

Comment (marekkirejczyk) [reply src/app/modals/CreateAccount.tsx:41]: ✅

File: src/app/modals/CreateAccount.tsx [hunk 2]
```diff
@@ -19,32 +19,59 @@ enum CreationStatus {
 
 export function CreateAccount({ onAccountCreated }: CreateAccountProps) {
   const [userName, setUserName] = useState<string>('');
-  const [status, setStatus] = useState<CreationStatus>(CreationStatus.NotStarted);
-  const [message, setMessage] = useState<string>('Deploying account...');
-  const {pxe} = usePXE();
-  const {faucet} = useDeveloperMode();
+  const [account, setAccount] = useState<AccountWalletWithPrivateKey | null>(null);
+  const { pxe } = usePXE();
```
[the remark this answers is not in the document]
Comment (marekkirejczyk) [reply src/app/modals/CreateAccount.tsx:23]: Yasss! Ticker added.

[the remark this answers is not in the document]
Comment (marekkirejczyk) [reply src/app/modals/CreateAccount.tsx]: Also fixed spaces :)

**Pull request merged**